projects/cashmere/src/lib/table/cell.ts

```diff
@@ -135,3 +135,18 @@ export class HcCell extends CdkCell {
         elementRef.nativeElement.classList.add(`hc-table-justify-` + columnDef.justify);
     }
 }
+
+/** Row index cell template container that adds the right classes and role. */
+@Directive({
+    selector: 'hc-index-cell, td[hc-index-cell], th[hc-index-cell]',
+    host: {
```

Comment:
I prefer consistency, so if this is what we're doing elsewhere, just disregard this, but the Angular style guide suggests to use the `@HostBinding` decorators instead of the metadata: https://angular.io/guide/styleguide#hostlistenerhostbinding-decorators-versus-host-metadata=

Reply:
Based on the reasonings provided, it's probably not that important since it doesn't really seem to apply here

Reply:
Hey @jonest - that's a great catch, and I'm pretty sure that code comes from back when this was all abstracted from Angular Material's table.  It looks like they haven't updated their syntax yet in their repo, but I agree - `@hostbinding` is what we use everywhere else in Cashmere.  I'll make the adjustment as a part of this PR.

Reply:
Only noticed because I didn't know what that stuff meant so I looked up the docs on it, haha 😆